02-express-tutorial/routes/people.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const router = express.Router();
+
+const {
+  addPerson,
+  getPeople,
+  getPersonById,
+  updatePerson,
+  deletePerson,
+} = require("../controllers/people");
+
+// create routers/people.js
+router.get("/", getPeople);
+router.post("/", addPerson);
+router.get("/:id", getPersonById);
+router.put("/:id", updatePerson);
+router.delete("/:id", deletePerson);
```

Comment:
Nitpick (doesn't require changes):
When logging a user out of a session, you'll typically use a POST instead of a DELETE request

Reply:
Nitpick (doesn't require changes):
When naming endpoints, I like to prefix them with an action, so it's easier to read/understand what the route expects to happen. 

For example:
```js
// instead of...
// http://localhost:3000/api/v1/people/6
router.delete('/:id', deletePerson);

// it becomes...
// http://localhost:3000/api/v1/people/delete/6
router.delete('/delete/:id', deletePerson);
```

Reply:
Okay, got it. I'll keep that in mind. Thank you.